misc/check_metfile.py

```diff
@@ -0,0 +1,64 @@
+# Script dat van een metfile leest en dan checkt of de profielinformatie en de metingen genoeg elementen hebben.
+# Dit geeft nu namelijk een fout bij het inlezen van de data voor Wetterskip
+# Geeft een lijst terug met daarin de profielnamen waarin een fout zit
+
+import os
+import csv
+path_input = 'K:\Tekeningen Amersfoort\\2018\TI18275 Wetterskip WIT missende profielen\Tekening\Bewerkingen\Uitzoeken_profielen_watergangen_versie_2\Voorbereiding_9okt\\van_mails_everts\\naverwerking\Inlezen_WIT'
+path_metfile = os.path.join(path_input,'cluster 15 versie 2.met')
+#path_metfile = os.path.join(path_input,'cluster 23 versie 3-aangepast.met')
+
+# List om alle informatie op te slaan
+info_list = []
+
+with open (path_metfile, 'r') as metfile:
+    print 'Metfile geopend: ', metfile
+
+    # Maakt de info als 1 lange text, handig om in te zoeken
+    met = metfile.read().replace('\n', '').replace('\r', '')
+
+    # Split de data op profielniveau
+    profielen = met.split('<PROFIEL>')
+
+    # Ga elk profiel af
+    for each in profielen[1:]:
+        profiel_elementen_totaal = each.split(',')
```

Comment:
profiel_elementen_totaal is een wel erg lange naam die volgens mij ook korter kan  (zeker omdat je hem regelmatig gebruikt). Ik zou er properties of elementen van maken